mne/evoked.py

```diff
@@ -159,12 +159,13 @@ def data(self, data):
         self._data = data
 
     @fill_doc
-    def get_data(self, picks=None, tmin=None, tmax=None):
+    def get_data(self, picks=None, units=None, tmin=None, tmax=None):
```

Comment:
This is a within-cycle change (you just added this in one of those last PRs), right? Just want to make sure we aren't breaking existing code that has been in a release like `.get_data(picks, tmin, tmax)`

Reply:
yup, get_data for Evoked is completely new --> and I did it this way, because it aligns with how units, tmin, tmax are ordered in Epochs.get_data

Reply:
Great, feel free to merge once CIs come back happy as this seems pretty straightforward!